IssueTriage.md

```diff
@@ -0,0 +1,103 @@
+# Triage Guidance for WebNN Repo
+
+TODO: Consider removing these, or fit these into the documentation below.
+
+- **enhancement** - unclear if this is about the spec text, the proposed API, etc. Prefer **feature request** etc.
+- **help wanted** - unclear what help is needed; use **needs PR** or **question** instead?
+
+
+## Label Usage
+
+Labels are used for:
+
+- Understanding the status of a specific issue and next steps to resolve it.
+- Understanding the scope of work remaining on broad efforts (e.g. aligning with best practices, fixing normative issues, etc)
+- Identifying areas to contribute.
+
+The working group chairs and spec editors should regularly review bugs and ensure that labels are accurate, and ensure that issues are getting appropriate attention; for example, scheduling discussion of new feature requests, discussion to resolve outstanding questions, and drawing attention to issues that are ready for a contributor to author a PR.
+
+
+## Types
+
+Every issue should have one of these issue types, and only rarely more than one.
+
+- **bug** - a flaw in the specification that will require a normative fix; for example, an algorithm computes an output incorrectly
+- **conventions** - where the spec does not conform to specification best practices from Web IDL, Bikeshed, Infra, etc.
+- **use case** - a new use case for the API that should be documented or considered; may spawn other issues
+- **process** - a meta issue about how the specification is evolved; for example, a discussion of issue labels
+- **testing** - discussion of test coverage
+- **feature request** - _PROPOSED_ - suggestion for an addition to the proposed API
+- **interop** - _PROPOSED_ - discussion of differences in behavior across implementations
+- **samples** - _PROPOSED_ - suggestions for additional or improved samples
+
+
+## Spec Impact
+
+These broad categories describe the projected impact on the specification and implementation of an issue.
+
+- **editorial** - spec text could be improved, but does not imply normative behavior changes to the proposed API
+- **normative** - _PROPOSED_ - resolving the issue would require behavior changes, likely affecting implementations and tests
+
+TODO: I'm not convinced of the utility of these; maybe drop them?
```

Comment:
One use case for "editorial" is to filter out issues that are https://www.w3.org/2023/Process-20231103/#editorial-change to assess whether the spec satisfies Rec Track transition requirements at various maturity stages.

I think we don't need normative.